lib/view/home_page.dart

```diff
@@ -1,68 +1,129 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
 import 'package:wh_covid19/hard_data.dart';
 import 'package:wh_covid19/style.dart';
 import 'package:wh_covid19/view/info_view.dart';
 import 'package:wh_covid19/widget/card_container.dart';
+import 'package:wh_covid19/utils/color.dart';
 
-class HomePage extends StatelessWidget {
+class HomePage extends StatefulWidget {
   @override
-  Widget build(BuildContext context) {
-    return Scaffold(
-      backgroundColor: AppColors.appBackground,
-      body: CustomScrollView(
+  _HomePageState createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  double _scrollPosition = 0;
+
+  Widget _renderBackgroundContainer(BuildContext context) {
+    var size = MediaQuery.of(context).size;
+    // Ensure that the image uses the full parent width
+    final Widget mainHeader = SvgPicture.asset(
+      'assets/images/main_header.svg',
+      width: size.width,
+    );
+    return Container(width: size.width, child: mainHeader);
+  }
+
+  /**
+   * Renders a stack with a single child node.
+   */
+  Widget _renderStack(
+    BuildContext context,
+    Widget child,
+  ) {
+    return Stack(
+      children: <Widget>[_renderBackgroundContainer(context), child],
+    );
+  }
+
+  Widget _renderList(BuildContext context) {
+    return SliverList(
+      delegate: SliverChildListDelegate([
+        Container(height: 12),
+        CardContainer(
+          title: 'Look After Yourself',
+          cards: staffWelfare,
+        ),
+        CardContainer(
+          title: 'Intubations',
+          cards: intubation,
+        ),
+        CardContainer(
+          title: 'ICU',
+          cards: icu,
+        ),
+        // Make sure the bottom CardContainer has room to breathe.
+        SizedBox(height: 12),
+      ]),
+    );
+  }
+
+  Widget _renderBody(BuildContext context) {
+    var appBarHeight = 70, logoHeight = 24;
```

Comment:
final instead of var, put each variable on a different line.

Reply:
yep! and applied to rest